client/state/themes/actions.js

```diff
@@ -460,9 +502,10 @@ export function installAndTryAndCustomize( themeId, siteId ) {
  * @param  {String}   siteId       Jetpack Site ID
  * @return {Function}              Action thunk
  */
-export function tryAndCustomize( themeId, siteId ) {
+export function tryAndCustomizeTheme( themeId, siteId ) {
 	return ( dispatch, getState ) => {
-		const theme = getTheme( getState(), siteId, themeId );
+		const siteIdOrWpcom = isJetpackSite( getState(), siteId ) ? siteId : 'wpcom';
```

Comment:
> ...since right now we need the theme object for getThemeCustomizeUrl... 

Aha, gotcha 👍 

> isJetpackSite( getState(), siteId ) && ! getTheme( getState(), siteId, themeId ) being a recurring pattern

Indeed. In a different PR.